src/components/ProjectSearch/ProjectSearch.jsx

```diff
@@ -0,0 +1,14 @@
+import React from 'react';
+import { Link } from 'react-router-dom';
+
+const ProjectSearch = () => {
+
+    return (
+        <>
+            <p>This is the Project Search page.</p>
+            <Link to="/home">Home</Link>
```

Comment:
Add links to correct pages... we can also update all this links later synchronously